storage poc tests failing

Client fails doing VirtioBlkGet:
```
2022/08/03 18:15:15 could not get VirtioBlk Controller: rpc error: code = Unavailable desc = error reading from server: EOF
exit status 1
```

From server:
```
2022/08/03 18:15:15 VirtioBlkGet: Received from client: controller_id:8
2022/08/03 18:15:15 Sending to SPDK: {"jsonrpc":"2.0","id":11,"method":"vhost_get_controllers","params":{"name":"VirtioBlk8"}}
2022/08/03 18:15:15 Received from SPDK: {11 {-32603 No such device} 0xc0000c8738}
2022/08/03 18:15:15 error: vhost_get_controllers: json response error: No such device
2022/08/03 18:15:15 Received from SPDK: []
2022/08/03 18:15:15 expecting exactly 1 result
panic: runtime error: index out of range [0] with length 0
```

--- a/storage/client/frontend.go
+++ b/storage/client/frontend.go
@@ -111,16 +111,11 @@ func do_frontend(conn grpc.ClientConnInterface, ctx context.Context) {
 
 	// VirtioBlk
 	c4 := pb.NewVirtioBlkServiceClient(conn)
-	rv1, err := c4.VirtioBlkCreate(ctx, &pb.VirtioBlkCreateRequest{Controller: &pb.VirtioBlk{Name: "VhostNvme8", Bdev:"Malloc1"}})
+	rv1, err := c4.VirtioBlkCreate(ctx, &pb.VirtioBlkCreateRequest{Controller: &pb.VirtioBlk{Name: "VirtioBlk8", Bdev:"Malloc1"}})
 	if err != nil {
 		log.Fatalf("could not create VirtioBlk Controller: %v", err)
 	}
 	log.Printf("Added: %v", rv1)
-	rv2, err := c4.VirtioBlkDelete(ctx, &pb.VirtioBlkDeleteRequest{ControllerId: 8})
-	if err != nil {
-		log.Fatalf("could not delete VirtioBlk Controller: %v", err)
-	}
-	log.Printf("Deleted: %v", rv2)
 	rv3, err := c4.VirtioBlkUpdate(ctx, &pb.VirtioBlkUpdateRequest{Controller: &pb.VirtioBlk{Name: "OPI-Nvme"}})
 	if err != nil {
 		log.Fatalf("could not update VirtioBlk Controller: %v", err)
@@ -141,4 +136,9 @@ func do_frontend(conn grpc.ClientConnInterface, ctx context.Context) {
 		log.Fatalf("could not stats VirtioBlk Controller: %v", err)
 	}
 	log.Printf("Stats: %v", rv6.Stats)
+	rv2, err := c4.VirtioBlkDelete(ctx, &pb.VirtioBlkDeleteRequest{ControllerId: 8})
+	if err != nil {
+		log.Fatalf("could not delete VirtioBlk Controller: %v", err)
+	}
+	log.Printf("Deleted: %v", rv2)
 }

--- a/storage/docker-compose.yml
+++ b/storage/docker-compose.yml
@@ -43,7 +43,7 @@ services:
     volumes_from:
       - spdk:rw
     volumes:
-      - ./:/app
+      - ./:/app:z
     networks:
       - opi
     command: go run jsonrpc.go frontend.go backend.go middleend.go server.go
@@ -52,7 +52,7 @@ services:
     image: golang:alpine
     working_dir: /app/client
     volumes:
-      - ./:/app
+      - ./:/app:z
     networks:
       - opi
     command: go run frontend.go backend.go middleend.go client.go -addr=opi-spdk-server:50051

